app/forms/field_handlers/field_handler.py

```diff
@@ -4,8 +4,8 @@
 from wtforms import validators, Field
 
 from app.data_model.answer_store import AnswerStore
-from app.questionnaire.location import Location
 from app.forms.validators import ResponseRequired, format_message_with_title
+from app.questionnaire.location import Location
```

Comment:
Any chance we could put Location into the Questionnaire __init__.py and change this to:

```suggestion
from app.questionnaire import Location
```

Reply:
Done